### DISCUSSION TITLE
Up for testing SCSI?

### DISCUSSION THREAD

**Discussion by mo-g** [geerlingguy/raspberry-pi-pcie-devices#27] [Ideas]
I've found a PCIe SCSI chipset which should have kernel support. Happy to send you one, but I can't quite stretch to sending you a disk array or such to go with it. Have any SCSI target devices lying around, and if so, up for trying this?

N.B. SCSI scsi. Not any of this newfangled UAS, SAS, FC junk.

**Comment by geerlingguy** [comment]
I have an old SCSI (1, not 2) hard drive I used to use in an external enclosure with my Mac. It would be interesting to see if it even still spins up!
